app/src/org/commcare/android/database/connect/models/ConnectJobDeliveryRecord.java

```diff
@@ -74,7 +74,7 @@ public static ConnectJobDeliveryRecord fromJson(JSONObject json, int jobId) thro
         delivery.lastUpdate = new Date();
 
         delivery.deliveryId = json.has(META_ID) ? json.getInt(META_ID) : -1;
-        delivery.date = json.has(META_DATE) ? ConnectNetworkHelper.parseDate(json.getString(META_DATE)) : new Date();
+        delivery.date = json.has(META_DATE) ? ConnectNetworkHelper.convertUTCToLocal(json.getString(META_DATE)) : new Date();
```

Comment:
I think the value that gets stored in the database should stay in UTC. The problem is that they may travel to another time zone later and we'll still want to know the correct dates. If we know the DB value is always UTC, we can always adjust it properly later.